argparse/argparse_galaxy_translation.py

```diff
@@ -164,7 +169,7 @@ def _StoreAction(self, param, tool=None):
 
         # Moved because needed in developing repeat CLI
         if positional:
-            flag_wo_dashes = 'positional_%s' % self.positional_count
+            flag_wo_dashes = '%s_%s' %(param.dest, self.positional_count)
```

Comment:
That definitely looks better. You could probably write some global counter and then incremenent every time you see a `param.dest` and if it's `>1` then add the number? But I'm not sure that would be a huge benefit? Something like

```python
from collections import Counter
params_seen = Counter()

....

v = params_seen[param.dest]
flag_wo_dashes = '%s%s' % (param.dest, '_' + str(v) if v > 0 else '')
params_seen[param.dest] += 1
```

Reply:
I reused the positional_count member for this .. OK?

Reply:
Looks fine to me.